mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -15,18 +15,22 @@ import dotty.tools.dotc.util.ParsedComment
 import org.eclipse.lsp4j.CompletionItemKind
 import org.eclipse.lsp4j.CompletionItemTag
 import org.eclipse.lsp4j.Range
+import org.eclipse.lsp4j.TextEdit
 
 sealed trait CompletionValue:
   def label: String
+  def insertText: Option[String] = None
   def snippetSuffix: Option[String] = None
+  def additionalEdits: List[TextEdit] = Nil
+  def range: Option[Range] = None
```

Comment:
I'm fine with adding these fields to the abstract `CompletionValue`. What do you think about this? @dos65 

Kinda off-topic: I'm wondering maybe we should always pre-compute the `additionalEdits` before creating `CompletionValue`s, because while `CompletionValue.Workspace` generate auto imports (as `additionalEdits` in `CompletionsProvider` after `CompletionValue`, some completions pre-compute auto imports before `CompletionValue`.

As a result, I am always confused about when and where should I generate the `additionalEdits`, and maybe it makes it harder to comprehend the program.

Reply:
I'm also fine with adding them at least for a while.
It feels like the completion processing is just happens in different places. For interpolater everything is calculated while others are processed at the moment of convertation them into lsp structure.

Reply:
It might be hard with the more custom completions to compute everything later on :thinking: